test/broadcast.jl

```diff
@@ -409,3 +409,12 @@ Base.Broadcast.broadcast_c(f, ::Type{Array19745}, A, Bs...) =
     @test isa(aa .+ 1, Array19745)
     @test isa(aa .* aa', Array19745)
 end
+
+# broadcast should only "peel off" one container layer
```

Comment:
Unfortunately I don't have much time for looking for better/simpler tests for today. Maybe we could revisit them another occasion?